client/src/components/visualisation/ConfigMenu.jsx

```diff
@@ -0,0 +1,253 @@
+import React, { PropTypes } from 'react';
+import DashSelect from '../common/DashSelect';
+import ColumnMenu from './configMenu/ColumnMenu';
+import LabelColumnMenu from './configMenu/LabelColumnMenu';
+import LabelInput from './configMenu/LabelInput';
+
+
+const getDatasetArray = datasetObject => {
+  const datasetArray = [];
+  const sortFunction = (a, b) => a.name.toLowerCase() > b.name.toLowerCase();
+
+  Object.keys(datasetObject).forEach(key => {
+    datasetArray.push(datasetObject[key]);
+  });
+
+  datasetArray.sort(sortFunction);
+
+  return datasetArray;
+};
+
+const getDatasetOptions = datasetArray => {
+  const output = [];
+
+  datasetArray.forEach(option => {
+    output.push({
+      value: option.id, label: option.name,
+    });
+  });
+
+  return output;
+};
+
+const getDashSelectOptionsFromColumnArray = array => {
+  const output = [];
+
+  if (array) {
+    array.forEach((entry, index) => {
+      output.push({
+        value: index, label: entry.title,
+      });
+    });
+  }
+  return output;
+};
+
+const getSubTitle = subtitle => (
+  <h3 className="subtitle">{subtitle}</h3>
+);
```

Comment:
Yes! Nice. You correctly intuited that the only reason I didn't make this a component in the first instance is that it is too trivial for a separate file. This syntax is very succinct for a small component 👍